apps/ai/src/tools/getEventTypes.ts

```diff
@@ -0,0 +1,57 @@
+import { DynamicStructuredTool } from "langchain/tools";
+import { z } from "zod";
+
+import { env } from "../env.mjs";
+import type { EventType } from "../types/eventType";
+import { context } from "../utils/context";
+
+/**
+ * Fetches event types by user ID.
+ */
+export const fetchEventTypes = async () => {
+  // TODO: comment this out once the bug is fixed with get event types in the API app
+  return [
+    {
+      id: 22,
+      length: 15,
+      title: "Quick Chat",
+    },
+  ];
```

Comment:
This should be removed. The getEventTypes call works in production, but it seems like there is an issue with getEventTypes in the main branch in the api when hosting locally. Once this issue is fixed, we can remove this temporary code and it should work.

Reply:
Removed!